### PR TITLE
Fix L4D2 Talker regarding uncommon infected

### PR DESCRIPTION
**Pull request by True-Boredom**
Fixes l4d2 survivors from being unable to call out uncommon infected on certain maps
Adds "norepeat" to the uncommon infected callouts that were missing it, to be consistent
Adds an Ellis story to c4m5, c13m2, c13m3, an c13m4

### CODE REVIEW

File: root/scripts/talker/coach.txt [hunk 1]
```diff
@@ -3326,6 +3326,14 @@ Rule EllisStoryInterruptGenericC4M4Coach
 	Response EllisStoryInterruptGenericCoach
 }
 
+Rule EllisStoryInterruptGenericC4M5Coach
```
Comment (xanaguy) [inline root/scripts/talker/coach.txt:3329]: Pointless. Generic rules will be used instead anyways.

File: root/scripts/talker/coach.txt [hunk 2]
```diff
@@ -3352,6 +3360,29 @@ Rule EllisStoryReac01Coach
 	Response EllisStoryReac01Coach
 }
 
+Rule EllisStoryInterruptC13M2Coach
```
Comment (xanaguy) [inline root/scripts/talker/coach.txt:3363]: Again, Generic rules can be used the same way.

File: root/scripts/talker/coach.txt [hunk 3]
```diff
@@ -4941,6 +4972,7 @@ Rule SurvivorSpottedWorldCloseCoach
 Response SurvivorSpottedArmoredCoach
 {
 	norepeat
+	scene "scenes/Coach/SeeArmored01.vcd"  //That zombie's got armor. I want armor!
```
Comment (xanaguy) [inline root/scripts/talker/coach.txt:4975]: Discussed with TrueBoredom. Will adjust in the next commit.

File: root/scripts/talker/mechanic.txt [hunk 4]
```diff
@@ -5056,6 +5069,44 @@ Rule EllisStoryReac01GoatMechanic
 	Response EllisStoryReac01GoatMechanic
 }
 
+Response C13M2EllisStoryMechanic
```
Comment (xanaguy) [inline root/scripts/talker/mechanic.txt:5072]: Can do a generic story rule with all stories instead, need Kerry's help first via a new criteria to filter out chapter 1 maps.

**Review by xanaguy — COMMENTED** [review]
Discussed with True Boredom. Will push a commit soon that will address some things within this PR.

File: root/scripts/talker/mechanic.txt (hunk 4 above)
Comment (True-Boredom) [reply root/scripts/talker/mechanic.txt:5072]: We _could_ do that, but I decided to use stories that had some context to the situation

**Comment by Tsuey** [comment]
Thank you for this!

Addressed [here](https://github.com/Tsuey/L4D2-Community-Update/commit/c8180fec1f90e4294a9f956a9f21ac1b0c8c4e67) and closing.

**Pull request closed without merging**